src/main/scala/jigg/pipeline/RegexDocumentAnnotator.scala

```diff
@@ -0,0 +1,30 @@
+package jigg.pipeline
+
+import java.util.Properties
+import scala.xml.Node
+
+class RegexDocumentAnnotator(override val name: String, override val props: Properties) extends Annotator {
+
+  @Prop(gloss = "Regular expression to segment documents (if omitted, specified regular expression is used)") var pattern = ""
```

Comment:
This comment (gloss) seems a bit odd. Probably description in () is unnecessary.

```Scala
@Prop(gloss = "Regular expression to segment documents") var pattern = """\n{2,}"""
```

then you can use `pattern` variable directory as `raw.split(pattern)`.